nutils/function.py

```diff
@@ -151,6 +151,7 @@ def prepare_eval(self, *, ndims: Optional[int] = None, opposite: bool = False, n
         a points axis.
     '''
 
+    warnings.deprecation('`prepare_eval` is deprecated; use `lower` instead')
```

Comment:
Shouldn't this wait until the arguments of `lower` are stable?

Reply:
done